app/models/crawler.rb

```diff
@@ -0,0 +1,8 @@
+class Crawler < ActiveRecord::Base
+
+  validates_presence_of :title, :desc
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.